ui/src/kapacitor/reducers/queryConfigs.js

```diff
@@ -119,6 +118,14 @@ const queryConfigs = (state = {}, action) => {
         [queryId]: nextQueryConfig,
       })
     }
+
+    case 'KAPA_REMOVE_FUNCS': {
+      const {queryID, fields} = action.payload
```

Comment:
in both of the cases above, this is `queryId` with a lowercase `d`. Here there's an uppercase `D`. Should be consistent

Reply:
I will go into the whole app and replace `Id` with `ID` or vice versa.  `ID` is currently most prevalent.